packages/firestore/src/platform_node/load_protos.ts

```diff
@@ -14,48 +14,23 @@
  * limitations under the License.
  */
 
-const dynamicRequire = require;
-const protobufjs = dynamicRequire('protobufjs');
-export type ProtobufProtoBuilder = any;
-
-import { NodeCallback, nodePromise } from '../util/node_api';
-
-export function loadProtosAsync(): Promise<ProtobufProtoBuilder> {
-  return nodePromise((callback: NodeCallback<ProtobufProtoBuilder>) => {
-    loadProtos(callback);
-  });
-}
+import * as grpc from 'grpc';
 
 /**
- * Loads the protocol buffer definitions for the datastore. This is a thin
- * wrapper around protobufjs.loadProtoFile which knows the location of the
- * proto files.
+ * Loads the protocol buffer definitions for Firestore.
  *
- * @param callback if specified, the load is performed asynchronously and
- *     the protos are supplied to the callback.
- * @returns the ProtoBuilder if the callback is unspecified.
+ * @returns The GrpcObject representing our protos.
  */
-export function loadProtos(
-  callback?: NodeCallback<ProtobufProtoBuilder>
-): ProtobufProtoBuilder | undefined {
-  const builder = protobufjs.newBuilder({
+export function loadProtos(): grpc.GrpcObject {
+  const options = {
     // Beware that converting fields to camel case does not convert the tag
     // fields in oneof groups (!!!).
```

Comment:
Does this refer to valueType? ProtobufJS 6.x+ is now doing this.

Reply:
Interesting.  I tried to go down the route of depending directly on protobufjs again and upgrading to 6.x, but it did not seem straightforward (the entire way we're consuming protobufjs has changed an doesn't seem to be documented in the release notes, etc.).  So I'm punting on this for now. I did update the comment to mention that upgrading to protobufjs 6.x would resolve this though.